common-arcana.lic

```diff
@@ -229,6 +229,7 @@ module DRCA
   def release_cyclics
     get_data('spells')
       .spell_data
+      .reject { |_name, properties| properties['persistent'] }
```

Comment:
❓ Are you also planning on marking spells as `persistent` in this PR?

Reply:
marked all spells as persistent: false in base-spells.yaml